src/app/forms/templates/psv/psv-brakes.template.ts

```diff
@@ -1,49 +1,67 @@
+import { ValidatorNames } from '@forms/models/validators.enum';
 import { FormNode, FormNodeTypes } from '../../services/dynamic-form.types';
 
-export const PsvBrakeSection: FormNode = {
+export const PsvBrakesTemplate: FormNode = {
   name: 'psvBrakesSection',
   label: 'Brakes',
-  type: FormNodeTypes.GROUP,
+  type: FormNodeTypes.SECTION,
   children: [
     {
       name: 'brakes',
       type: FormNodeTypes.GROUP,
       children: [
         {
           name: 'brakeCode',
-          label: 'Brake Code',
+          label: 'Brake code',
           value: '',
           type: FormNodeTypes.CONTROL,
+          validators: [{ name: ValidatorNames.Required }]
```

Comment:
Should be 6 char max length

Reply:
That was omitted intentionally, as we are not allowing for free text, rather we are using the autocomplete field. Just for you, I still included the validator 🙂